Vagrantfile

```diff
@@ -26,23 +32,21 @@ Vagrant.configure("2") do |config|
     end
   end
 
-  config.vm.define "freebsd" do |c|
-    c.ssh.shell = "csh"
-    c.vm.box = "freebsd/FreeBSD-10.2-RELEASE"
-
+  config.vm.define "freebsd11" do |c|
+    c.vm.box = "freebsd/FreeBSD-11.1-RELEASE"
+    c.vm.guest = :freebsd
+    c.vm.hostname = "freebsd11"
+    c.vm.base_mac = "6658695E16F0"
     c.vm.network "private_network", type: "dhcp"
-    c.vm.synced_folder ".", "/vagrant", type: "nfs"
+    c.vm.synced_folder ".", "/vagrant", disabled: true
```

Comment:
the upstream Vagrantfile clones /vagrant (which is the shared folder) to /home/vagrant/crystal. i'm on nixos here and getting nfs folder sharing to work is a PITA. would it be ok to introduce an environment variable that disables folder sharing? 
 
so i would set VAGRANT_DISABLE_SHARED to true and the box comes up cleanly in my case. the default doesn't change. 

i will also look into making ${O} configurable, but will open a second issue for this.